File: frontend/src/app/model/order-info.ts
export class OrderInfo {
  totalPrice: number;
  totalQuantity: number;
  status: string;

  constructor(totalPrice: number, totalQuantity: number, status: string) {
    this.totalPrice = totalPrice;
    this.totalQuantity = totalQuantity;
    this.status = status;
  }
}

File: backend/core-service/src/main/java/com/ecommerce/core/kafka/event/order/OrderInfo.java
package com.ecommerce.core.kafka.event.order;

import java.io.Serial;
import java.io.Serializable;
import java.math.BigDecimal;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OrderInfo implements Serializable {
    @Serial
    private static final long serialVersionUID = -677352479002726461L;
    private BigDecimal totalPrice;
    private Integer totalQuantity;
    private String status;

    @Override
    public String toString() {
        return "OrderInfoDTO{" +
                "totalPrice=" + totalPrice +
                ", totalQuantity=" + totalQuantity +
                ", status='" + status + '\'' +
                '}';
    }
}
